slack_sdk/web/client.py

```diff
@@ -2209,387 +3264,864 @@ def openid_connect_token(
+        return self.api_call("reactions.add", params=kwargs)
 
-    def reactions_get(self, **kwargs) -> SlackResponse:
-        """Gets reactions for an item."""
+    def reactions_get(
+        self,
+        *,
+        channel: Optional[str] = None,
+        file: Optional[str] = None,
+        file_comment: Optional[str] = None,
+        full: Optional[bool] = None,
+        timestamp: Optional[str] = None,
+        **kwargs,
+    ) -> SlackResponse:
+        """Gets reactions for an item.
+        https://api.slack.com/methods/reactions.get
+        """
+        kwargs.update(
+            {
+                "channel": channel,
+                "file": file,
+                "file_comment": file_comment,
+                "full": full,
+                "timestamp": timestamp,
+            }
+        )
         return self.api_call("reactions.get", http_verb="GET", params=kwargs)
 
-    def reactions_list(self, **kwargs) -> SlackResponse:
-        """Lists reactions made by a user."""
+    def reactions_list(
+        self,
+        *,
+        count: Optional[int] = None,
+        cursor: Optional[str] = None,
+        full: Optional[bool] = None,
+        limit: Optional[int] = None,
+        page: Optional[int] = None,
+        team_id: Optional[str] = None,
+        user: Optional[str] = None,
+        **kwargs,
+    ) -> SlackResponse:
+        """Lists reactions made by a user.
+        https://api.slack.com/methods/reactions.list
+        """
+        kwargs.update(
+            {
+                "count": count,
+                "cursor": cursor,
+                "full": full,
+                "limit": limit,
+                "page": page,
+                "team_id": team_id,
+                "user": user,
+            }
+        )
         return self.api_call("reactions.list", http_verb="GET", params=kwargs)
 
-    def reactions_remove(self, *, name: str, **kwargs) -> SlackResponse:
+    def reactions_remove(
+        self,
+        *,
+        name: str,
+        channel: Optional[str] = None,
+        file: Optional[str] = None,
+        file_comment: Optional[str] = None,
+        timestamp: Optional[str] = None,
+        **kwargs,
+    ) -> SlackResponse:
         """Removes a reaction from an item.
-
-        Args:
-            name (str): Reaction (emoji) name. e.g. 'thumbsup'
+        https://api.slack.com/methods/reactions.remove
         """
-        kwargs.update({"name": name})
-        return self.api_call("reactions.remove", json=kwargs)
+        kwargs.update(
+            {
+                "name": name,
+                "channel": channel,
+                "file": file,
+                "file_comment": file_comment,
+                "timestamp": timestamp,
+            }
+        )
+        return self.api_call("reactions.remove", params=kwargs)
 
-    def reminders_add(self, *, text: str, time: str, **kwargs) -> SlackResponse:
+    def reminders_add(
```

Comment:
There is an unusual options field for `recurrence` which is not as well documented. Though from investigating it looks like it would accept a string. Did you omit on purpose for now?

Reply:
Thanks. I just missed the one. Added with optional str type